### PR TITLE
Docs: alert for the possibility of manipulating logits

### PR DESCRIPTION
**Pull request by gante**
# What does this PR do?

As @osanseviero told me on slack, popular logit manipulation strategies like `min_p` are a bit obscure in our docs.

A proper fix will come with the [`generate` refactor](https://github.com/huggingface/transformers/issues/30810), where a diagram of what `generate` does will be added. In that diagram, there will be a dedicated section for logits processors.

Meanwhile, a quick patch: this PR adds a tip in the decoding strategies section alerting for the existence and importance of logit manipulation.

### CODE REVIEW

**Comment by gante** [comment]
doc builder screenshot: 
Screenshot 2024-08-06 at 14 50 06

**Review by ArthurZucker — APPROVED** [review]
Thanks

**Comment by osanseviero** [comment]
Nice, thanks!
